charts/filtered-resource-service/templates/statefulset.yaml

```diff
@@ -80,15 +80,9 @@ spec:
           value: {{ $item.value }}
         {{- end }}
         volumeMounts:
-        - name: {{ include "filtered-resource-service.classpathJars.name" . }}
-          mountPath: {{ include "filtered-resource-service.classpathJars.mount" . }}
         - name: config-volume
           mountPath: {{ include "filtered-resource-service.config.path" . }}
       volumes:
-      - name: {{ include "filtered-resource-service.classpathJars.name" . }}
-        persistentVolumeClaim:
-          claimName: {{ include "filtered-resource-service.classpathJars.name" . }}
-          readOnly: true
```

Comment:
Is there a reason that these have been removed for the FRS?

Reply:
Not needed, plus s3 akka-http conflicts with the version used in frs